go/cmd/dolt/commands/diff.go

```diff
@@ -46,13 +47,17 @@ import (
 )
 
 const (
-	SchemaOnlyDiff    = 1
-	DataOnlyDiff      = 2
+	SchemaOnlyDiff = 1
+	DataOnlyDiff   = 2
+	Summary        = 4
```

Comment:
Probably should declare these as binary literals since they used as bit masks

Reply:
fixed